src/journal-remote/journal-remote.h

```diff
@@ -33,6 +33,7 @@ struct MHDDaemonWrapper {
         struct MHD_Daemon *daemon;
 
         sd_event_source *event;
+        sd_event_source *timerfd;
```

Comment:
please name this differently, as "timerfd" is kinda misleading, since this does not directly translate to a timer fd...

to avoid ambiguitty maybe rename the existing "event" field to "io_event" or so, and then your new "timerfd" field to "timer_event" or so.